src/menu/menu.js

```diff
@@ -1,81 +1,136 @@
 import { fetchData } from "../helpers.js";
 
+export function getRestaurantsDetailsTemplate() {
+  return `
+  <div id="loader" class="center"></div>
+  <h1>Restaurant Details</h1>
+  <div>
+    <h4 id="name"></h4>
+    <h4 id="address1-label" style="display: inline-block;">Address I: </h4>
+    <p id="address1" style="display: inline-block;"></p>
+    <br>
+    <h4 id="address2-label" style="display: inline-block;">Address II: </h4>
+    <p id="address2" style="display: inline-block;"></p>
+    <br>
+    <h4 id="latitude-label" style="display: inline-block;">Latitude:</h4>
+    <p id="latitude" style="display: inline-block;"></p>
+    <br>
+    <h4 id="longitude-label" style="display: inline-block;">Longitude:</h4>
+    <p id="longitude" style="display: inline-block;"></p>
+    <h1>Menu</h1>
+  </div>
+ `;
+}
+
 export function getMenuTemplate() {
-  return ` <h1>Menu</h1>`;
+  return `
+  <table>
+  <tbody id="main"></tbody>
+  <tfoot>
+    <tr id="total-row-quantity">
+      <td colspan="2" id="total-quantity">Total quantity: 0</td>
+    </tr>
+    <br>
+    <tr id="total-row-price">
+      <td colspan="2" id="total-price">Total price: $0.00</td>
+    </tr>
+  </tfoot>
+  </table>
+  `;
+}
+
+export function getRestaurantDetailsByID(id) {
+  fetchData(
+    `https://private-anon-7231255228-pizzaapp.apiary-mock.com/restaurants/${id}`
+  )
+    .then((result) => {
+      document.getElementById("name").innerHTML = result.name;
+      document.getElementById("address1").innerHTML = result.address1;
+      document.getElementById("address2").innerHTML = result.address2;
+      document.getElementById("latitude").innerHTML = result.latitude;
+      document.getElementById("longitude").innerHTML = result.longitude;
+      getRestaurantMenu();
+    })
+    .catch((error) => {
+      console.error(error);
+    });
 }
 
 export function getRestaurantMenu(id) {
   fetchData(
     `https://private-anon-7231255228-pizzaapp.apiary-mock.com/restaurants/${id}/menu?category=Pizza&orderBy=rank`
```

Comment:
Also, we should make the two API calls at the same time. For this you can use `Promise.all()`.